source/includes/usage-examples/sample-app-intro.rst

```diff
@@ -0,0 +1,18 @@
+.. TODO: change link from quick start primer to subscriber implementation page
+
+You can use the following sample application to test the code examples on this
+page. To use the sample application, perform the following steps:
+
+1. Ensure you have the {+driver-short+} installed.
+#. Create a new Java project in your IDE.
+#. Create a package called ``helpers`` within the existing ``java`` package in
+   your project.
+#. Create a new Java file in the ``helpers`` package
+   named ``SubscriberHelpers``. Implement the ``Subscriber`` implementations, which are
```

Comment:
Q: I'm not an expert in subscriber implementations, but is it normal to say "implement the implementations"?